tests/Database/SelectFields/InterfaceTests/CommentType.php

```diff
@@ -20,12 +19,7 @@ public function fields(): array
     {
         $interface = GraphQL::type('LikableInterface');
 
-        return [
-                'title' => [
-                    'type' => Type::nonNull(Type::string()),
-                    'alias' => 'title',
-                ],
-            ] + $interface->getFields();
+        return $interface->getFields();
```

Comment:
Is this change actually related somehow or just a cleanup?

Reply:
Just some cleanup :)